bucket/qqmusic.json

```diff
@@ -0,0 +1,46 @@
+{
+    "homepage": "https://y.qq.com/",
+    "description": "freemium music streaming services by Tencent",
+    "version": "1873",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://y.qq.com/i/user_terms.html"
+    },
+    "url": "https://dldir1.qq.com/music/clntupate/QQMusic_Setup_1873.exe#/dl.7z",
+    "hash": "md5:73e1bb2821a4531e1960df58be0765fb",
+    "installer": {
+        "script": [
+            "# simulate instok check",
+            "$helperScript = $(Join-Path $(Find-BucketDirectory -Root -Name dorado) scripts/qqmusic/instok)",
+            "Copy-Item -Path $helperScript -Destination $dir",
+            "# handle runtime cache",
+            "Import-Module $(Join-Path $(Find-BucketDirectory -Root -Name dorado) scripts/DoradoUtils.psm1)",
```

Comment:
It is fine to import packages from *Dorado*, but we should copy the script into the *Extras* bucket as well.

Reply:
Sorry for not understanding what you meant.
Please feel free to correct me and modify the script, in order to make this package better ^^

Reply:
`$helperScript = $(Join-Path $(Find-BucketDirectory -Root -Name dorado) scripts/qqmusic/instok)`

This means the installer script will load the file `(SCOOP DIRECTORY)\buckets\dorado\scripts\qqmusic\instok`.

This will cause 2 problems:
**(1)**: Users will have to add `dorado` bucket first, otherwise there will be a **"file not found "** error, which is not convienient and looks confusing to users.
**(2)**: In the future, if  *Dorado* bucket's maintainers rename the file (or, change the file structure of the bucket). The script will not work because of the changes in *Dorado*.


So, what you should do is to replace `Dorado` with `Extras`, and include files like `instok` and `DoradoUtils.psm1` in this PR.

Reply:
Dorado's script `Mount-ExternalRuntimeData` will create a junction between the data folder in `$env:APPDATA` and `$persist_dir`, but Extras prefers not. ([Issue #8647](https://github.com/ScoopInstaller/Extras/issues/8647#issuecomment-1180074323))